cmd/eksctl-anywhere/cmd/supportbundle.go

```diff
@@ -97,7 +109,7 @@ func (csbo *createSupportBundleOptions) createBundle(ctx context.Context, since,
 	}
 	defer close(ctx, deps)
 
-	supportBundle, err := deps.DignosticCollectorFactory.DiagnosticBundle(clusterSpec, deps.Provider, getKubeconfigPath(clusterSpec.Cluster.Name, csbo.wConfig), bundleConfig)
+	supportBundle, err := deps.DignosticCollectorFactory.DiagnosticBundle(clusterSpec, deps.Provider, getKubeconfigPath(clusterSpec.Cluster.Name, csbo.clusterKubeconfig), bundleConfig)
```

Comment:
What happens if this is passed in as empty? Would it be able to get the right kubeconfig to get the bundle?

Reply:
https://github.com/aws/eks-anywhere/blob/d975bda33508cae68d1368f66f0e61370d1ae538/cmd/eksctl-anywhere/cmd/common.go#L46
If no kubeconfig is specified in cmd, it use default path to locate kubeconfig file, which is the same path upon creation. The bug was reproduced by move or rename the kubeconfig file/dir. So if customers have to provide kubeconfig file path if they change it, otherwise it only use the default path